app/src/main/java/com/skgtecnologia/sisem/domain/model/footer/FooterModel.kt

```diff
@@ -3,5 +3,6 @@ package com.skgtecnologia.sisem.domain.model.footer
 import com.skgtecnologia.sisem.domain.model.body.ButtonModel
 
 data class FooterModel(
-    val buttonModelList: List<ButtonModel>
+    val leftButton: ButtonModel?,
+    val rightButton: ButtonModel?
```

Comment:
This is the one I want to discuss with the others

Probably we should always expect one button, either left or right